src/content/reference/react-dom/components/common.md

```diff
@@ -985,12 +988,12 @@ textarea { display: block; margin-top: 5px; margin-bottom: 10px; }
 
 </Sandpack>
 
-To see why rendering arbitrary HTML is dangerous, replace the code above with this:
+Untuk mengetahui mengapa merender HTML sewenang-wenang itu berbahaya, ganti kode di atas dengan ini:
```

Comment:
```suggestion
Untuk mengetahui mengapa me-*render* HTML sewenang-wenang itu berbahaya, ganti kode di atas dengan ini:
```

Reply:
noted and fixed, thank you for the review